objcgen/operatoroverloads.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using IKVM.Reflection;
+using Embeddinator;
+
+namespace ObjC
+{
+	public static class OperatorOverloads
+	{
+		struct OperatorInfo
+		{
+			public string MetadataName { get; set; }
+			public string FriendlyName { get; set; }
+			public int ArgumentCount { get; set; }
+
+			public OperatorInfo (string metadataName, string friendlyName, int argumentCount)
+			{
+				MetadataName = metadataName;
+				FriendlyName = friendlyName;
+				ArgumentCount = argumentCount;
+			}
+
+			public bool Contains (string name) => MetadataName == name || FriendlyName == name;
+		}
+
+		// From https://msdn.microsoft.com/en-us/library/ms229032(v=vs.110).aspx and https://msdn.microsoft.com/en-us/library/8edha89s.aspx
+		// with ambigious elements and non-overridable elements removed
+		static List<OperatorInfo> OperatorMapping = new List<OperatorInfo>() { 
+			new OperatorInfo ("op_Addition", "Add", 2),
+			new OperatorInfo ("op_Subtraction", "Subtract", 2),
+			new OperatorInfo ("op_Multiply", "Multiply", 2),
+			new OperatorInfo ("op_Division", "Divide", 2),
```

Comment:
The doc (https://msdn.microsoft.com/en-us/library/8edha89s.aspx) suggested:

Mod or Remainder

But what do we do if the user has both? In the other cases they listed just one option.